### DISCUSSION TITLE
Community Development Around Plugins

### DISCUSSION THREAD

**Discussion by danielhelfand** [vmware-tanzu/octant#468] [Plugins]
What are the thoughts at this time around developing plugins and contributing them so that others can use these plugins? How would one go about using a plugin that has been contributed?

**Comment by wwitzel3** [comment]
We are encouraging folks to use the `octant-plugin` topic for their repos. See: https://github.com/topics/octant-plugin

As of right now we don't maintain a curated list of "official" octant plugins. This maybe something we will look in to doing as we get closer to 1.0.

**Comment by danielhelfand** [comment]
@wwitzel3 Thanks! I'll check out what's available under the topic.

**Comment by wwitzel3** [comment]
We added a page to https://reference.octant.dev to list user contributed plugins, https://reference.octant.dev/?path=/docs/docs-plugins-a1-list-of-known-plugins--page

You can add to that list by opening a PR against the file referenced: `File: web/src/stories/docs/plugins/AppxListKnownPlugins.story.mdx`

**Reply by wwitzel3** [reply]
Currently there are none listed, so if you know of any, I encourage you to make a PR to add them :)
